pre-commit.sh

```diff
@@ -1,4 +1,17 @@
 #!/usr/bin/env bash
 set -e
 
-./docker-gradle clean test
+./docker-gradle clean -x test
+
+./startup.sh
+
+echo "Waiting 60 seconds for services."
+sleep 60
```

Comment:
Is this a temporary work around?

Reply:
Temporary workaround until we implement healthchecks in the services that will tell us when they're up and working.